pyvista/core/pointset.py

```diff
@@ -27,6 +29,10 @@
 
 log = logging.getLogger(__name__)
 log.setLevel('CRITICAL')
+DEFAULT_INPLACE_WARNING = 'You did not specify a value for `inplace` and the ' \
+    'defualt for value will be changing to `False` in future versions for ' \
+    'point-based meshes (e.g., `PolyData`). Please make sure you are not ' \
+    'assuming this to be an inplace operation.'
```

Comment:
Typo, and I'd strongly recommend using anything other than explicit line continuations.
```suggestion
DEFAULT_INPLACE_WARNING = 'You did not specify a value for `inplace` and the ' \
    'default for value will be changing to `False` in future versions for ' \
    'point-based meshes (e.g., `PolyData`). Please make sure you are not ' \
    'assuming this to be an inplace operation.'
```

Reply:
Codespell should have picked up on this. I'm actually pretty sure `make codespell` does nothing

Reply:
I had codespell complain to me, but I wouldn't be surprised if that only applied to docstrings. This is a random string in a module.